src/main/java/org/fenixedu/academic/ui/struts/action/administrativeOffice/lists/StudentListByDegreeDA.java

```diff
@@ -343,6 +330,13 @@ private void fillSpreadSheetFilters(SearchStudentsByDegreeParametersBean searchB
                 spreadsheet.addHeader(statute.getName().getContent());
             }
         }
+        spreadsheet.newHeaderRow();
+        if (searchBean.hasAnyProgramConclusion()) {
+            spreadsheet.addHeader(getResourceMessage("label.programConclusions") + ":");
+            for (ProgramConclusion programConclusion : searchBean.getProgramConclusions()) {
+                spreadsheet.addHeader(programConclusion.getName().getContent());
+            }
+        }
```

Comment:
I used the same pattern that is used for statutes, since that header only contains filter information, where should I add the cell values?